department-of-veterans-affairs/va.gov-cms
# VACMS-3813: disable select admin menu items

## Description

See #3813 + #3847

## Testing done

manual . not adding automated tests for this one

## Screenshots

<img width="1042" alt="Screen Shot 2021-01-04 at 12 03 31 PM" src="https://user-images.githubusercontent.com/16477724/103570590-a07c0a80-4e86-11eb-9dff-3b9be611e9ff.png">
<img width="766" alt="Screen Shot 2021-01-04 at 12 03 39 PM" src="https://user-images.githubusercontent.com/16477724/103570712-d6b98a00-4e86-11eb-8f2f-1da0ef12ed4e.png">

## QA steps

1. [x] login and inspect "Sections" menu at the top of admin toolbar. Confiirm that "Sections > Veteran Health Administration > VAMC Facilities" menu item does not have any children
1. [x] Inspect "Content" menu at the bottom level of admin toolbar. Confirm that all menu items under "Content > Menus" are disabled and dont have any children. _[NOTE: since top level menu links under Content > Menus are content, not config, I will manually delete them after this PR is deployed ]_
1. [x] Confirm that Login Performance test time benchmark is set 3 seconds and tests are passing

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [x] Yes, and it's written in issue #3813  and queued for publication.
